include/wil/com.h

```diff
@@ -3325,6 +3325,79 @@ WI_NODISCARD auto make_range(IEnumXxx* enumPtr)
 #endif // WIL_HAS_CXX_17
 #endif // WIL_ENABLE_EXCEPTIONS
 
+#if WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP | WINAPI_PARTITION_SYSTEM)
+#ifdef __WIL_WIN32_HELPERS_INCLUDED

```

Comment:
Is there any potential issue with just including `win32_helpers.h` in this file? `com.h` is not set up to be includable more than once, making patterns like this very brittle and prone to include order issues.

Reply:
I'm not sure.  I was hoping you and other experts could advise on that.  The filetime helpers are definitely useful in this case so I'd prefer to use them instead of doing ULARGE_INTEGER math manually.

Reply:
Unless there are good, specific concerns, I'd say just include the header. If problems arise, the number of failures will determine how we might go about mitigating the issue (e.g. something like an opt-out for few failures). Not that I anticipate that, though. Risking problems is better than introducing a ticking time bomb of include order dependencies.

Reply:
That makes sense.  I'll make that change in a future iteration.